app/views/ChildBenefitView.scala.html

```diff
@@ -32,11 +32,26 @@
 
     <h1 @if(!isIvrRedirect){class="header-margin-top"}>@messages("child.benefit.title")</h1>
 
+<!--Add back in on the 4th Jan-->
+<!--<div class="govuk-inset-text">-->
+<!--        <p>@messages("global.opening.times.title")</p>-->
+<!--        <p>@messages("global.opening.times.8to8")</p>-->
+<!--        <p>@messages("global.opening.times.saturday.8to4")</p>-->
+<!--        <p>@messages("global.closed.time.sun.bankholidays")</p>-->
+<!--    </div>-->
+
+<!--Remove on the 4th Jan-->
     <div class="govuk-inset-text">
         <p>@messages("global.opening.times.title")</p>
```

Comment:
Does this field need to be changed to `Christmas and New Year opening times:` on the updated pages?